### PR TITLE
ETCM-3756 Rename readme to DEVELOPMENT.md, create standard readme.

### PR DESCRIPTION
**Pull request by NeilBurgess42**
Also changed Mantis to SC_EVM or sc-evm as appropriate

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,29 +1,18 @@
...
-- [Node.js](https://nodejs.org/en/) - tested with v12
-  - (optional) [nvm](https://github.com/nvm-sh/nvm) - you can use nvm to automatically pick-up the Node version you need
-- [yarn](https://classic.yarnpkg.com/en/) - tested with v1.22.5
+As of [insert date], IOG will no longer be updating or maintaining this repo.
 
-## Initialize the project
+After three months of experimentation with the proof-of-concept EVM sidechain testnet, we have gathered valuable learnings from this experience and this innovative playground. New use cases and functionality were tested, feedback from the community was gathered, and we are now shifting our focus back to the strategic evolution of our sidechains approach.  
+  
+All information included below is considered publicly available and is free to be leveraged by developers to fork it to build and experiment with their own EVM sidechain solution. Should you have any questions, please reach out to our team on our IOG Discord server.  
+## 2. Description
```
Comment (jb-lunatech) [inline README.md:11]: nit: no newline around here. Sometimes, markdown is interpreted differently with or without newline

**Review by jb-lunatech — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1,29 +1,18 @@
-# Mantis Faucet Web
+# SC_EVM Faucet
 
-Web interface for Mantis Faucet
```
Comment (drospa) [inline README.md:3]: Web faucet for EVM sidechain

Comment (NeilBurgess42) [reply README.md:3]: This is not the current line 3. Current line 3 in DEVELOPMENT.md:
Web interface for SC_EVM Faucet
Current line 3 in README.md is blank.

Comment (drospa) [reply README.md:3]: @NeilBurgess42 we shall explain this is a web interface of the faucet for EVM sidechain in the initial README

Comment (NeilBurgess42) [reply README.md:3]: OK

**Pull request merged**